.travis.yml

```diff
@@ -4,18 +4,19 @@ dist: trusty
 
 branches:
   except:
-  - staging.tmp
+    - staging.tmp
 
 cache:
   cargo: true
-
-addons:
-  chrome: stable
+  directories:
+    - node_modules
```

Comment:
we should cache `$HOME/.npm` instead (see https://github.com/ember-cli/ember-new-output/blob/v3.0.2/.travis.yml#L14 and https://stackoverflow.com/questions/42521884/should-i-have-travis-cache-node-modules-or-home-npm)

Reply:
TIL… 😁
Changed!